src/App.tsx

```diff
@@ -15,71 +16,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface AppState {
+  selectedGood: string;
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, AppState> {
+  state: AppState = {
+    selectedGood: 'Jam',
```

Comment:
Replace with `goods[goods.indexOf('Jam')] || goods[0]`

In your code - if you don't have the element 'Jam' on the goods array - it will make bugs.